test/specs/test.e2e.apply.claim.js

```diff
@@ -100,5 +100,6 @@ describe("apply and claim journey for Sheep", () => {
     await title.waitForDisplayed({ timeout: 5000 });
```

Comment:
I dont think this is needed anymore. And Im also not sure we need to await the title on the line above?